dockerfiles/docker-libv4l-build-cross-armhf/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM rust:1.36.0-buster
```

Comment:
There is no need for having (version specific) rust from the beginning.

Reply:
Sure. Just kept it as is.
Will change into `latest`.